.github/workflows/integration_test.yml

```diff
@@ -16,11 +16,11 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     env:
-      ROLLKIT_VERSION: "v1.0.0-beta.2"
+      EVNODE_VERSION: "v1.0.0-beta.2.0.20250818133040-d096a24e7052"
       IGNITE_VERSION: "v29.3.1"
-      IGNITE_ROLLKIT_APP_VERSION: "rollkit/v0.3.0"
-      ROLLKIT_IMAGE_REPO: "rollkit-gm"
-      ROLLKIT_IMAGE_TAG: "latest"
+      IGNITE_EVOLVE_APP_VERSION: "main" # use tagged when apps has tagged (blocked on things)
```

Comment:
The app won't be tagged until evnode and ev-abci are tagged again so Ignite keeps providing a great UX and does not have a released version using pseudo commits..